verification/test-wdls/TestImputationBeagle.wdl

```diff
@@ -20,7 +21,7 @@ workflow TestImputationBeagle {
       Array[String] contigs # list of possible contigs that will be processed. note the workflow will not error out if any of these contigs are missing
       String reference_panel_path_prefix # path + file prefix to the bucket where the reference panel files are stored for all contigs
       String genetic_maps_path # path to the bucket where genetic maps are stored for all contigs
-      String output_basename # the basename for intermediate and output files
+      String output_basename # the basename for intermediate and output filesFloat min_dr2_for_inclusion
```

Comment:
missing a newline here?

Reply:
haha its actually above, i must have tried to delete this and failed. removing